### PR TITLE
API-14985 Added patient James Beck and associated records.

### PR DESCRIPTION
**Pull request by taylorrassmannlibertyits**
https://vajira.max.gov/browse/API-14985

NOTE: I kept the files in pretty json format for review. I will squish them before final merge.

### CODE REVIEW

File: datamart/dm-records-1012643432V477452/dmMedOrd1012643432477452000O.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+  "objectType": "MedicationOrder",
+  "objectVersion": "1",
+  "cdwId": "1012643432477452000:O",
+  "patient": {
+    "type": "Patient",
+    "reference": "1012643432V477452",
+    "display": "James M. Beck"
+  },
+  "dateWritten": "2021-11-12T01:43:31Z",
```
Comment (jblefkove-va) [inline datamart/dm-records-1012643432V477452/dmMedOrd1012643432477452000O.json:10]: Is the full timestamp intentional?

File: datamart/dm-records-1012643432V477452/dmMedOrd1012643432477452001O.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  "objectType": "MedicationOrder",
+  "objectVersion": "1",
+  "cdwId": "1012643432477452001:O",
+  "patient": {
+    "type": "Patient",
+    "reference": "1012643432V477452",
+    "display": "James M. Beck"
+  },
+  "dateWritten": "2021-11-06T01:43:31Z",
+  "status": "DISCONTINUED",
```
Comment (jblefkove-va) [inline datamart/dm-records-1012643432V477452/dmMedOrd1012643432477452001O.json:11]: completed?

File: datamart/dm-records-1012643432V477452/dmMedOrd1012643432477452002O.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+  "objectType": "MedicationOrder",
+  "objectVersion": "1",
+  "cdwId": "1012643432477452002:O",
+  "patient": {
+    "type": "Patient",
+    "reference": "1012643432V477452",
+    "display": "James M. Beck"
+  },
+  "dateWritten": "2021-11-12T01:43:31Z",
```
Comment (jblefkove-va) [inline datamart/dm-records-1012643432V477452/dmMedOrd1012643432477452002O.json:10]: ditto

File: datamart/dm-records-1012643432V477452/dmMedOrd1012643432477452001O.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+  "objectType": "MedicationOrder",
+  "objectVersion": "1",
+  "cdwId": "1012643432477452001:O",
+  "patient": {
+    "type": "Patient",
+    "reference": "1012643432V477452",
+    "display": "James M. Beck"
+  },
+  "dateWritten": "2021-11-06T01:43:31Z",
```
Comment (jblefkove-va) [inline datamart/dm-records-1012643432V477452/dmMedOrd1012643432477452001O.json:10]: ditto

File: datamart/dm-records-1012643432V477452/dmMedOrd1012643432477452000O.json (hunk 1 above)
Comment (taylorrassmannlibertyits) [reply datamart/dm-records-1012643432V477452/dmMedOrd1012643432477452000O.json:10]: Nope! Must have been leftover copy. Setting to midnight

**Review by jblefkove-va — APPROVED** [review]

**Pull request merged**